### PR TITLE
added the services page

### PR DESCRIPTION
**Pull request by hashim-rana-hub**

### CODE REVIEW

File: assets/scss/_import.scss [hunk 1]
```diff
@@ -2,3 +2,4 @@
 @import './mixins';
 @import './variable';
 @import './style';
+@import './typography';
```
Comment (hashim-rana-hub) [inline assets/scss/_import.scss]: add line at the end

File: assets/scss/_style.scss [hunk 2]
```diff
@@ -8,103 +8,91 @@ body {
...
+        linear-gradient($bg-dark,$bg-dark) padding-box,
+        linear-gradient(to right, #20D4FE 0%, #B722FF 100%) border-box;
       }
+      &:hover{
+        @include gradient;
+          &::after {
+            display: block;
+          }
+        }
+        }
```
Comment (hashim-rana-hub) [inline assets/scss/_style.scss]: indentation

File: assets/scss/_style.scss [hunk 3]
```diff
@@ -8,103 +8,91 @@ body {
...
 .services {
-  margin-top:4.5rem;
+  margin-top:72px;
   h3 {
-    @include h-3;
-  }
-  h4 {
-    color: $primary;
-    @include h-4;
     span{
```
Comment (hashim-rana-hub) [inline assets/scss/_style.scss:50]: add whitespace before "{"

File: assets/scss/_style.scss [hunk 4]
```diff
@@ -115,158 +103,137 @@ ul {
     align-items: baseline;
```
Comment (hashim-rana-hub) [inline assets/scss/_style.scss:103]: recheck

File: assets/scss/_style.scss [hunk 5]
```diff
@@ -115,158 +103,137 @@ ul {
...
     @include a-tag;
     color: $primary;
-    margin: 2.5rem 0rem 0rem;
+    margin: 40px 0px 0px;
     &:hover {
-      color: $secondary;
+      // color: $secondary;
```
Comment (hashim-rana-hub) [inline assets/scss/_style.scss]: remove commented code

File: assets/scss/_style.scss [hunk 6]
```diff
@@ -115,158 +103,137 @@ ul {
...
+      margin-left: .3125rem;
     }
   }
 }
 .about-us {
-  margin-top: 128px;
+  margin-top: 8rem;
   .first-row {
     align-items: center;
     .col-left {
```
Comment (hashim-rana-hub) [inline assets/scss/_style.scss:122]: rename class name

File: assets/scss/_style.scss [hunk 7]
```diff
@@ -115,158 +103,137 @@ ul {
...
+    .hours-annually {
       margin-top: 20px;
     }
   }
   .set-us-apart {
     margin-top: 5rem;
-    h4{
-      margin-top: 0px;
-      margin-bottom: 4.375rem;
+    h3{
```
Comment (hashim-rana-hub) [inline assets/scss/_style.scss:150]: add white space

File: assets/scss/_style.scss [hunk 8]
```diff
@@ -115,158 +103,137 @@ ul {
...
+      margin-top: 0rem;
+      margin-bottom: 70px;
     }
     .set-us-apart-listing {
       .set-us-apart-item {
         display: flex;
-        margin-left: 16px;
+        margin-left: 1rem;
         &:not(:last-child){
```
Comment (hashim-rana-hub) [inline assets/scss/_style.scss:158]: same as above

File: assets/scss/_style.scss [hunk 9]
```diff
@@ -115,158 +103,137 @@ ul {
...
   }
 }
 .contact-us {
-  margin-top: 7.8rem;
-  p {
-    @include p;
-    margin-top: 40px;
-    max-width: 22.5rem;
+  margin-top: 10rem;
+  padding: 0px 5rem;
```
Comment (hashim-rana-hub) [inline assets/scss/_style.scss]: refactor

File: assets/scss/_style.scss [hunk 10]
```diff
@@ -115,158 +103,137 @@ ul {
...
+  .description {
+    margin-top: 2.5rem;
+    max-width: 360px;
   }
   .information {
     display: grid;
-    h6 {
-      @include h-6;
-    }
     a {
```
Comment (hashim-rana-hub) [inline assets/scss/_style.scss:196]: apply generic styling seperatly

File: assets/scss/_style.scss [hunk 11]
```diff
@@ -115,158 +103,137 @@ ul {
...
       @include a-tag;
-
-      span {
-        font-size: 1.125rem;
-        font-weight: 400;
-      }
     }
   }
   .contact-detail {
```
Comment (hashim-rana-hub) [inline assets/scss/_style.scss:200]: contact-details

File: assets/scss/_style.scss [hunk 12]
```diff
@@ -115,158 +103,137 @@ ul {
...
 .footer {
-border-top: .0625rem solid #383838;
-margin-top: 48px;
-padding-top: 48px;
-padding-bottom: 32px;
-  p {
-    font-size: 1.125rem;
-    font-weight: 400;
-  }
+border-top: 1px solid #383838;
```
Comment (hashim-rana-hub) [inline assets/scss/_style.scss]: indentation

File: assets/scss/_style.scss [hunk 13]
```diff
@@ -115,158 +103,137 @@ ul {
...
+padding: 2.5rem 4.375rem;
   ul {
     display: flex;
     justify-content: flex-end;
     list-style-type: none;
     li:not(:last-child) {
-      margin-right: 1.5rem;
+      margin-right: 24px;
     } 
     li a {
```
Comment (hashim-rana-hub) [inline assets/scss/_style.scss]: apply nesting

File: assets/scss/_typography.scss [hunk 14]
```diff
@@ -0,0 +1,35 @@
+h1,h2,h3,h4,h5,h6,p{
```
Comment (hashim-rana-hub) [inline assets/scss/_typography.scss]: add whitespace before "{"

File: assets/scss/_typography.scss [hunk 15]
```diff
@@ -0,0 +1,35 @@
...
+  font-weight: 700;
+}
+p{
+  font-size: 18px;
+  font-weight: 400;
+}
+// button{
+//   font-size: 18px;
+//   font-weight: 500;
+// }
```
Comment (hashim-rana-hub) [inline assets/scss/_typography.scss]: add line at end

File: pages-scss/_about-us.scss [hunk 16]
```diff
@@ -37,19 +34,15 @@
...
-    font-size: 24px;
+  h1{
     font-weight: 600;
   }
-  p {
-    @include p;
+  h6 {
+    font-weight: 600;
   }
   .we-are-strong {
```
Comment (hashim-rana-hub) [inline pages-scss/_about-us.scss]: refactor class-name

File: pages-scss/_about-us.scss [hunk 17]
```diff
@@ -37,19 +34,15 @@
...
     align-items: center;
     margin-bottom: 4rem;
-    h2 {
-      @include h-2;
-    }
   }
   .ceo-sec {
```
Comment (hashim-rana-hub) [inline pages-scss/_about-us.scss]: ceo-sec ---> ceo-details

File: pages-scss/_about-us.scss [hunk 18]
```diff
@@ -60,10 +53,6 @@
       width: 120px;
       height: 120px;
     }
-    h4 {
-      font-size: 38px;
-      font-weight: 700;
-    }
     .ceo-detail {
```
Comment (hashim-rana-hub) [inline pages-scss/_about-us.scss]: rename

File: pages-scss/_about-us.scss [hunk 19]
```diff
@@ -82,16 +71,9 @@
 }
 .our-work {
```
Comment (hashim-rana-hub) [inline pages-scss/_about-us.scss]: projects-snapshot

File: pages/index-about-us.html [hunk 20]
```diff
@@ -215,68 +223,129 @@ <h6>We Know How To Scale</h6>
...
     </div>
     <div class="container our-work">
-      <div class="snapshot">
-        <h3>Here Is A Quick Snapshot
-          <span>Of What We Have Done So Far</span>
-        </h3>
-      </div>
+      <h5>Here Is A Quick Snapshot</h5>
+      <h3>Of What We Have Done So Far</h3>
       <div class="snapshot-projects">
```
Comment (hashim-rana-hub) [inline pages/index-about-us.html]: snapshot-projects ===> projects-list

File: pages-scss/_about-us.scss [hunk 21]
```diff
@@ -109,41 +91,37 @@
...
-      margin: 5rem 0rem;
-      span {
-        @include span;
-        color: $primary;
-        -webkit-text-fill-color: #ffffff;
-      }
     }
   }
-  .benifits-list {
+  .benifits-detail{
```
Comment (hashim-rana-hub) [inline pages-scss/_about-us.scss]: benefits

File: pages-scss/_about-us.scss [hunk 22]
```diff
@@ -109,41 +91,37 @@
...
     margin-top: 6.25rem;
-    .benifits-detail {
+    .benifits-list{
       display: flex;
-      flex-direction: column;
-      .title {
+      justify-content: space-between;
+      list-style-type: none;
+      width: 100%;
+      &:not(:last-child){
```
Comment (hashim-rana-hub) [inline pages-scss/_about-us.scss]: same as above

File: pages-scss/_services.scss [hunk 23]
```diff
@@ -0,0 +1,92 @@
+h2 {
+	text-align: left;
+}
+.product-scale {
+	margin-top: 107px;
+	margin-bottom: 130px;
```
Comment (hashim-rana-hub) [inline pages-scss/_services.scss]: indentation

File: pages-scss/_services.scss [hunk 24]
```diff
@@ -0,0 +1,92 @@
...
+      text-align: left !important;
+      font-size: 52px;
+      font-weight: 600;
+    }
+    p {
+      font-size: 20px;
+      font-weight: 400;
+      text-align: left;
+      // margin-bottom: 20px;
+      // margin-top: 16px;
```
Comment (hashim-rana-hub) [inline pages-scss/_services.scss]: remove commented code

File: assets/scss/_mixins.scss [hunk 25]
```diff
@@ -3,8 +3,8 @@
   -webkit-background-clip: text;
   -webkit-text-fill-color: transparent;
 }
-@mixin button{
-  background: linear-gradient(90deg, #20D4FE 0.09%, #B722FF 99.91%);
+@mixin button {
+  background: linear-gradient(90deg, #20D4FE 0%, #B722FF 100%);
   border:none;
 }
 @mixin a-tag {
```
Comment (Eyyan-Amir) [inline assets/scss/_mixins.scss:10]: style anchor tag not use mixin

File: assets/scss/_mixins.scss [hunk 26]
```diff
@@ -13,43 +13,12 @@
...
   font-weight: 700px;
   display: block;
 }
-@mixin truncate($lines,
-  $lh: 1.2,
-  $spacer:0) {
+@mixin truncate($lines,$lh: 1.2,$spacer:0) {
   line-height: $lh;
   max-height: $lh * $lines + $spacer + em;
   // overflow: hidden;
```
Comment (Eyyan-Amir) [inline assets/scss/_mixins.scss:24]: remove commented code

File: assets/scss/_style.scss [hunk 27]
```diff
@@ -8,103 +8,91 @@ body {
...
-      border:1px solid transparent;
-      background:
-      linear-gradient($bg-dark,$bg-dark) padding-box,
-      linear-gradient(to right, #20D4FE 0%, #B722FF 100%) border-box;
-    }
-    &:hover{
-      @include gradient;
-        &::after {
-          display: block;
+  padding: 30px;
```
Comment (Eyyan-Amir) [inline assets/scss/_style.scss:13]: match padding from designs.

**Pull request closed without merging**